src/marathon/kafka-scheduler.json

```diff
@@ -0,0 +1,39 @@
+{
+  "id": "/kafka-mesos-scheduler",
+  "cmd": "$(find jdk* -maxdepth 0 -type d)/bin/java -jar kafka-mesos-0.10.1.0-SNAPSHOT-kafka_2.11-0.10.2.0.jar scheduler --debug true --master=zk://mesos-master1:2181,mesos-master2:2181,mesos-master3:2181/mesos --zk=zk1-for-kafka:2181,zk2-for-kafka:2181,zk3-for-kafka:2181/KafkaCluster --api=http://$HOSTNAME:7000 --storage=zk:/kafkaStorage --user=kafka",
```

Comment:
I'm not a big fan of the hard coded jar names everywhere (esp to snapshot), I dont know really anything about marathon but is there a way to parameterize these?  Or maybe just have the command discover the jar automatically?

Alternatively, maybe we should just put this whole file in /examples (which I realize doesn't exist yet).